pom.xml

```diff
@@ -259,8 +274,48 @@
         <exclusion><groupId>org.locationtech.geogig</groupId><artifactId>geogig-web-api</artifactId></exclusion>
         <exclusion><groupId>org.locationtech.geogig</groupId><artifactId>geogig-osm</artifactId></exclusion>
         <exclusion><groupId>com.jolbox</groupId><artifactId>bonecp</artifactId></exclusion>
-      </exclusions>
-    </dependency>    
+      </exclusions -->
+    </dependency>
+	<dependency>
+      <groupId>ch.qos.logback</groupId>
+      <artifactId>logback-core</artifactId>
+      <version>${logback.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>ch.qos.logback</groupId>
+      <artifactId>logback-classic</artifactId>
+      <version>${logback.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.locationtech.geogig</groupId>
```

Comment:
Roger,

thanks Erik.


Best Regards,
Alessio Fabiani.

==
GeoServer Professional Services from the experts!
Visit http://goo.gl/it488V for more information.
==

Ing. Alessio Fabiani
@alfa7691
Founder/Technical Lead

GeoSolutions S.A.S.
Via di Montramito 3/A
55054  Massarosa (LU)
Italy
phone: +39 0584 962313
fax:     +39 0584 1660272
mob:   +39 331 6233686

http://www.geo-solutions.it
http://twitter.com/geosolutions_it

-------------------------------------------------------

*AVVERTENZE AI SENSI DEL D.Lgs. 196/2003*

Le informazioni contenute in questo messaggio di posta elettronica e/o
nel/i file/s allegato/i sono da considerarsi strettamente riservate. Il
loro utilizzo è consentito esclusivamente al destinatario del messaggio,
per le finalità indicate nel messaggio stesso. Qualora riceviate questo
messaggio senza esserne il destinatario, Vi preghiamo cortesemente di
darcene notizia via e-mail e di procedere alla distruzione del messaggio
stesso, cancellandolo dal Vostro sistema. Conservare il messaggio stesso,
divulgarlo anche in parte, distribuirlo ad altri soggetti, copiarlo, od
utilizzarlo per finalità diverse, costituisce comportamento contrario ai
principi dettati dal D.Lgs. 196/2003.



The information in this message and/or attachments, is intended solely for
the attention and use of the named addressee(s) and may be confidential or
proprietary in nature or covered by the provisions of privacy act
(Legislative Decree June, 30 2003, no.196 - Italy's New Data Protection
Code).Any use not in accord with its purpose, any disclosure, reproduction,
copying, distribution, or either dissemination, either whole or partial, is
strictly forbidden except previous formal approval of the named
addressee(s). If you are not the intended recipient, please contact
immediately the sender by telephone, fax or e-mail and delete the
information in this message that has been received in error. The sender
does not give any warranty or accept liability as the content, accuracy or
completeness of sent messages and accepts no responsibility  for changes
made after they were sent or for other risks which arise as a result of
e-mail transmission, viruses, etc.

---------------------------------------------------------------------

On Mon, Jul 11, 2016 at 4:21 PM, Erik Merkle <notifications@github.com>
wrote:

> In pom.xml
> <https://github.com/GeoNode/geoserver-geonode-ext/pull/46#discussion_r70266531>
> :
>
> > -      </exclusions>
> > -    </dependency>
> > +      </exclusions -->
> > +    </dependency>
> > +	<dependency>
> > +      <groupId>ch.qos.logback</groupId>
> > +      <artifactId>logback-core</artifactId>
> > +      <version>${logback.version}</version>
> > +    </dependency>
> > +    <dependency>
> > +      <groupId>ch.qos.logback</groupId>
> > +      <artifactId>logback-classic</artifactId>
> > +      <version>${logback.version}</version>
> > +    </dependency>
> > +    <dependency>
> > +      <groupId>org.locationtech.geogig</groupId>
>
> @afabiani <https://github.com/afabiani> If you change the dependency of
> gs-geogig to the GeoServer artifact, you may not need to explicitly add
> these GeoGig dependencies.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/GeoNode/geoserver-geonode-ext/pull/46/files/83415916ea580384facfeb342b4e975046f9fea5#r70266531>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABOARchD9QihL227sJgfejSTELbA1kn7ks5qUlGBgaJpZM4JJQeu>
> .
>